packages/create-llama/templates/types/streaming/nextjs/app/api/chat/route.ts

```diff
@@ -1,9 +1,12 @@
+import { initObservability } from "@/shared/observability";
```

Comment:
Isn't everythin in `/app` in NextJS?
```suggestion
import { initObservability } from "@/app/shared/observability";
```

Reply:
No, the app directory has a very [strict structure](https://nextjs.org/docs/app/building-your-application/routing/defining-routes) since it's mapped to URLs. Other files should sit outside of that directory